meorg_client/cli.py

```diff
@@ -286,8 +286,9 @@ def cli_analysis():
 # Add file commands
 cli_file.add_command(file_list)
 cli_file.add_command(file_upload)
-# cli_file.add_command(file_upload_parallel)
 cli_file.add_command(file_attach)
+cli_file.add_command(file_detach_all)
```

Comment:
Removed comment, added the detach_all option. This must have gotten lost somewhere in the flurry of commits.